pulser/simulation/simulation.py

```diff
@@ -179,15 +187,17 @@ def _build_basis_and_op_matrices(self):
             basis = ['r', 'g', 'h']
             projectors = ['gr', 'hg', 'rr', 'gg', 'hh']
 
-        self.basis = {b: qutip.basis(self.dim, i) for i, b in enumerate(basis)}
+        self.basis = {b: qutip.basis(self.dim, i)
+                      for i, b in enumerate(basis)}
```

Comment:
Why did you break this one into two lines? Did you try typing `self.basis` and then gave up?

Reply:
I think it was showing `line too long` error.